npe2/manifest/schema.py

```diff
@@ -42,6 +43,12 @@
 
 ENTRY_POINT = "napari.manifest"
 
+# The first release of npe2 defines the first engine version: 0.0.0.
+# As the contract around plugins evolve the ENGINE_NUMBER should be
+# increased follow SemVer rules. Note that sometimes the version number
+# will change even though no npe2 code changes.
+ENGINE_VERSION = "0.0.0"
```

Comment:
thoughts on `0.0.0` instead of `1.0.0` or `0.1.0` or `0.0.1` ?

Reply:
I go back and forth. I like 0.1.0 too.

Reply:
```suggestion
ENGINE_VERSION = "0.1.0"
```